packages/components/psammead-media-indicator/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 4.0.0 | [PR#3062](https://github.com/bbc/psammead/pull/3062) Remove spacing on the low-level component |
```

Comment:
👍 Making it as a major version is good. We should update the Readme - the `topStory` prop being removed, that we expect users of this component to add their own spacing where relevant. 
That children passed include a Timestamp passed in. We shouldn't need the DateTime component defined here, with the specific spacing, when spacing may change depending on context.  https://github.com/bbc/psammead/blob/f8537de4a42cedfee4f1135c965946303dc3c88f/packages/components/psammead-media-indicator/src/index.jsx#L70